Neos.Flow/Classes/Property/TypeConverter/StringConverter.php

```diff
@@ -76,7 +76,7 @@ class StringConverter extends AbstractTypeConverter
     /**
      * @var array<string>
      */
-    protected $sourceTypes = ['string', 'integer', 'float', 'bool', 'array', \DateTimeInterface::class];
+    protected $sourceTypes = ['string', 'integer', 'float', 'boolean', 'array', \DateTimeInterface::class];
```

Comment:
I think only this was the actual breaking part of the original change, because the TypeConverter map will look up by the normalized type of the source, and hence this TypeConverter would no longer match for `boolean`. On top, the above `canConvertFrom()` method probably also would need to be changed to sth. like `TypeHandling::normalizeType($methodParameter['type']) === TypeHandling::normalizeType(gettype($source));`.

Reply:
Yes, I think so too. But I couldn't confirm because no test failed so I'd rather revert this completely and then re-apply it cleanly (and possibly to a different branch)